docs/en/commands/status.md

```diff
@@ -0,0 +1,79 @@
+# Status
+
+Retrieve the server’s current status.
+
+## Example Usage
+
+```java
+// Java
+// new Status(); // TODO: How to get status in Java
+```
+```python
+# Python 
+# (not implemented)
+```
+```javascript
+// Javascript
+// webdriver.io
+driver.status();
+
+// wd
+await driver.status();
+```
+
+## Description
+
+Returns information about whether a remote end is in a state in which it can create new sessions and can additionally include arbitrary meta information that is specific to the implementation.
+
+The readiness state is represented by the ready property of the body, which is false if an attempt to create a session at the current time would fail. However, the value true does not guarantee that a New Session command will succeed.
+
+Implementations may optionally include additional meta information as part of the body, but the top-level properties ready and message are reserved and must not be overwritten.
+
+## Client Docs
+
+* [Java](http://seleniumhq.github.io/selenium/docs/api/java/index.html)
+* [Python](http://selenium-python.readthedocs.io/api.html#selenium.webdriver.common.utils.is_url_connectable)
+* [Javascript (WebdriverIO)](http://webdriver.io/api/protocol/status.html)
+* [Javascript (WD)](https://github.com/admc/wd/blob/master/lib/commands.js#L44)
+
```

Comment:
* [Ruby](http://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/DriverExtensions/HasRemoteStatus#remote_status-instance_method)

---

^ is for now.(because it's marked `private`. So, I'll define in appium client side soon.)